src/librustc_parse/lexer/mod.rs

```diff
@@ -326,6 +326,50 @@ impl<'a> StringReader<'a> {
         suffix_start: BytePos,
         kind: rustc_lexer::LiteralKind,
     ) -> (token::LitKind, Symbol) {
+        //(This change will make sense in the next commit.)
```

Comment:
Thanks. I didn't know this was possible.
I didn't undo moving the code to the top though.

Reply:
For future readers, it would be better to minimize the diff.
I would be also better to cleanup the git history a bit because right now some commits do things that are then undone by later commits.

Reply:
Ok, I can do that.
Is there a better way than to manually redo the changes and commit inbetween?

Reply:
Rebasing is totally ok, we don't have a policy against it here.
I usually use `git rebase -i` that allows to reorder and squash commits automatically, and perhaps stop and modify them manually.